recipes/esys-escript/meta.yaml

```
@@ -0,0 +1,73 @@
+
+package:
+    name: esys-escript
+    version: 5.4
+source:
+    - svn_url: https://svn.geocomp.uq.edu.au/svn/esys13/trunk
+      svn_rev: 6850
+      folder: escript
+build:
+    skip: True  # [osx]
+    skip: True  # [win]
+    number: '0'
+    rpaths:
+        - ${PREFIX}/bin
+        - ${PREFIX}/include
+        - ${PREFIX}/lib
+        - ${SP_DIR}
+requirements:
+    build:
+        - {{ compiler('cxx') }}
+        - svn
+        - scons 3.0.5 py27_0
+        - cmake
+        - make
+        - openmp
+    host:
+        - boost-cpp 1.70.0 ha2d47e9_0
```

Please only specify versions, best is to actually only specify upto minor and leave out the patch version. This is just an example for here, please also adjust all others.
```suggestion
        - boost-cpp 1.70
```